File: Week01_Sep14/01_Beginner1/StefanMendoza/Beginner1.java
/**
 * File: Beginner1.java
 * Author: Stefan Mendoza
 * Date: 2 September 2016
 *
 * Week 1 / Beginner 1 for KU Programming Club (2016-2017)
 */

import java.util.Scanner;

public class Beginner1 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.println("What's your favorite color?");
        System.out.print("> ");
        String favoriteColor = input.nextLine();

        System.out.println("Your's is " + favoriteColor + "? That's a great color! Mine's red.");
    }
}

File: Week01_Sep14/01_Beginner1/StefanMendoza/Beginner1.cpp
/**
 * File: Beginner1.cpp
 * Author: Stefan Mendoza
 * Date: 2 September 2016
 *
 * Week 1 / Beginner 1 for KU Programming Club (2016-2017)
 */

#include <iostream>
#include <string>

int main() {
    std::cout << "What's your favorite color?" << std::endl;
    std::cout << "> ";
    std::string favoriteColor;
    std::cin >> favoriteColor;
    std::cout << "Your's is " << favoriteColor
    << "?  That's a great color! Mine's red." << std::endl;

    return 0;
}
